core/src/main/scala/org/apache/spark/deploy/PythonRunner.scala

```diff
@@ -74,6 +77,16 @@ object PythonRunner {
     // Launch Python process
     val builder = new ProcessBuilder((Seq(pythonExec, formattedPythonFile) ++ otherArgs).asJava)
     val env = builder.environment()
+    if (!Utils.isLocalRemote(sparkConf)) {
+      // For non-local remote, pass configurations to environment variables so
+      // Spark Connect client sets them. For local remotes, they will be set
+      // via Py4J.
+      // For PySpark specifically, we can't send other configurations through properties.
+      // So, here we should send it together.
+      val confs = sparkConf.getAll.filter(p =>
```

Comment:
Where are these confs being used? In client or on the server side. You don't have to pass them if they are needed on server side.

Reply:
This specific code path is only used in Spark Connect client (so I only pass `spark.sql.*`)